app/forms/idv/api_image_upload_form.rb

```diff
@@ -205,6 +205,15 @@ def submit_attempts
       rate_limiter.attempts if document_capture_session
     end
 
+    def processed_selfie_attempts_data
+      return {} if document_capture_session.nil? || !liveness_checking_required
+
+      captured_result = document_capture_session&.load_result
+      processed_selfie_count = (selfie_image_fingerprint ? 1 : 0)
```

Comment:
no need for the parens IMO
```suggestion
      processed_selfie_count = selfie_image_fingerprint ? 1 : 0
```